pkg/clustermanager/applier.go

```diff
@@ -20,10 +20,12 @@ const (
 	applyClusterSpecTimeout           = 2 * time.Minute
 	waitForClusterReconcileTimeout    = time.Hour
 	retryBackOff                      = time.Second
-	waitForFailureMessageErrorTimeout = 20 * time.Second
+	waitForFailureMessageErrorTimeout = time.Minute
 	defaultFieldManager               = "eks-a-cli"
 )
 
+var defaultConditionCheckTotalCount = 20
```

Comment:
I wanted to make this configurable for unit test.

Reply:
Can you make this a const? You can't update this from unit tests either since it's in a diff package

Reply:
If you want to make it configurable, pls add another member to the applier struct like with the other params